images/windows/entrypoint/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG BASE_IMAGE=mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
Do we still need these? We should be building entrypoint and nop using ko now, right?

Reply:
Those are the base images that are fed into the `ko` config.
I believe they are used today in the build pipeline - see https://github.com/tektoncd/pipeline/blob/main/tekton/build-push-ma-base-image.yaml

Reply:
Uhm, it looks like it's not used here though https://github.com/tektoncd/pipeline/blob/4f8c227794abd6b2157856892415d5c207352295/tekton/publish.yaml#L96-L100

Reply:
But we still run it as part of the release pipeline: https://github.com/tektoncd/pipeline/blob/4f8c227794abd6b2157856892415d5c207352295/tekton/release-pipeline.yaml#L107-L127

Reply:
Ok, so summarise, this is the `ko` config for `nop` and `entrypoint`:

https://github.com/tektoncd/pipeline/blob/4f8c227794abd6b2157856892415d5c207352295/tekton/publish.yaml#L107-L108

Where the `$COMBINED_BASE_IMAGE` is the result of:

https://github.com/tektoncd/pipeline/blob/4f8c227794abd6b2157856892415d5c207352295/tekton/publish.yaml#L97-L100

so it doesn't depend on anything else indeed.
The base images for `git-init` and `pullrequest-init`:

https://github.com/tektoncd/pipeline/blob/4f8c227794abd6b2157856892415d5c207352295/tekton/publish.yaml#L111-L113

Are based on the respective base images that are created out of the dockerfiles in `images/git-init` and `images/pullrequest-init`.

So, yeah, we don't need those dockerfiles anymore, and if we can use the latest release pipeline to build earlier releases, we should be able to avoid backport 🎉